engine/src/main/java/io/camunda/zeebe/engine/processing/processinstance/ProcessInstanceMigrationMigrateProcessor.java

```diff
@@ -103,6 +112,21 @@ public void processRecord(final TypedRecord<ProcessInstanceMigrationRecord> comm
         processInstanceKey, ProcessInstanceMigrationIntent.MIGRATED, value, command);
   }
 
+  @Override
+  public ProcessingError tryHandleError(
+      final TypedRecord<ProcessInstanceMigrationRecord> command, final Throwable error) {
+    if (error instanceof final UnsupportedElementMigrationException e) {
+      final String reason =
+          "Expected to migrate process instance '%s' but it contains an active element that is unsupported: %s"
```

Comment:
❌  The syntax is a bit blurry with indicating the element id. Would be better if we make the element id more explicit by saying:

`"Expected to migrate process instance '%s' but it contains an active element that is unsupported. Element id: %s"`

or something similar.